dashboardv3/public/js/views/detail_page/DetailPageLayoutView.js

```diff
@@ -176,15 +176,15 @@ define(['require',
                         typeName = Utils.getName(collectionJSON, 'typeName'),
                         superTypes = Utils.getNestedSuperTypes({ data: this.activeEntityDef.toJSON(), collection: this.entityDefCollection }),
                         isLineageRender = _.find(superTypes, function(type) {
-                            if (type === "DataSet" || type === "Process") {
+                            if (type === "Catalog" || type === "Process") {
                                 if (type === "Process") {
                                     isProcess = true;
                                 }
                                 return true;
                             }
                         });
                     if (!isLineageRender) {
-                        isLineageRender = (typeName === "DataSet" || typeName === "Process") ? true : null;
+                        isLineageRender = (typeName === "Catalog" || typeName === "Process") ? true : null;
```

Comment:
isLineageRender = (typeName === "Catalog" || typeName === "DataSet" || typeName === "Process") ? true : null;

@nikhilbonte21 Instead of changing Dataset to Catalog, add Catalog in if case so that inbuilt model also will also work.

Reply:
This is to be in sync with backend changes